src/ripple.js

```diff
@@ -78,7 +78,7 @@ var Ripple = {
             // No need to set positioning because ripple should be child of target and to it's relative position.
             // rippleContainer.style.left    = left + (((window.pageXOffset || document.scrollLeft) - (document.clientLeft || 0)) || 0) + "px";
             // rippleContainer.style.top     = top + (((window.pageYOffset || document.scrollTop) - (document.clientTop || 0)) || 0) + "px";
-            rippleContainer.style.width   = width + "px";
+            rippleContainer.style.width   = "calc(100% + " + border * 2 + "px)";
```

Comment:
it's ok for now, but in future it can break we use different `border-width` on alef and right ;)

Reply:
you already mentioned that in https://github.com/bethinkpl/vue-ripple-directive/pull/2#discussion_r1011351652 :+1: